ctl/import.go

```diff
@@ -439,12 +440,12 @@ func (cmd *ImportCommand) importValues(ctx context.Context, vals []pilosa.FieldV
 
 	// Group vals by slice.
 	logger.Printf("grouping %d vals", len(vals))
-	valsBySlice := pilosa.FieldValues(vals).GroupBySlice()
+	valsBySlice := http.FieldValues(vals).GroupBySlice()
```

Comment:
this appears to only be used in this command - we should move it over and unexport (after this is merged)

Reply:
ah no - client.go wasn't expanded, so I couldn't see it when searching